docs/index.md

```diff
@@ -0,0 +1,35 @@
+![Squish - version](https://img.shields.io/badge/Squish-v6.7.0+-brightgreen)
+![PyPI - Python Version](https://img.shields.io/pypi/pyversions/squape)
+![PyPI - License](https://img.shields.io/pypi/l/SQUAPE)
+# SQUAPE - Squish API Python Extension
+Python package that extends Squish Python API. It provides tools for everyday automated test cases development.
+
+### Requirements
+The minimal requirement for the SQUAPE package is Squish version 6.7.0. Individual features may require a more recent Squish package. 
+
+### Installation
+The package is available on the Python Package Index (PyPI).
+You can install it via `pip` with the following commands:
+```sh
+pip install squape
+```
+
+### Content
+The package consists of several modules:
+- report - adds features to enhance the reporting capabilities of Squish
+- settings - makes using various test settings easier and cleaner
+- video - adds features to enhance video capture capabilities of Squish
+- vps - extension of Squish verification points
```

Comment:
object_tree module should be mentioned here as well

Reply:
added :)